templates/nginx_internal.conf

```diff
@@ -21,29 +21,19 @@ http {
     # Learn more about nginx maps here http://nginx.org/en/docs/http/ngx_http_map_module.html
     map "$request_method:$http_accept" $proxpass {
         # If no explicit matches exists below, send traffic to lemmy-ui
-        default "http://lemmy-ui";
+        default "http://lemmy-ui:1234";
```

Comment:
So these lines were all missing the port? Not sure why it worked before. 

Maybe we should change all the docker containers to listen on standard http port (80), would make things easier than these random port numbers.